aptos-move/move-examples/drand/sources/lottery_test.move

```diff
@@ -119,9 +125,6 @@ module drand::lottery_test {
         //
         // Send a TXN with `drand_signed_bytes` to close the lottery and determine the winner
         //
-        let winner_addr = option::extract(&mut lottery::close_lottery(drand_signed_bytes));
-
-        debug::print(&string::utf8(b"The winner is: "));
-        debug::print(&winner_addr)
+        lottery::close_lottery(drand_signed_bytes);
```

Comment:
Why remove this? We need this when testing.

Reply:
Entry functions can't return values so we can't access the winner from `close_lottery`. We could make another function to return it but this would require refactoring the code to store the winner in a lottery instance.

Reply:
(This is why the code previously compiled with `cargo test` but not `move compile` - for whatever reason the Rust testing harness doesn't check for this)

Reply:
I just updated the contract with a public function that returns the current lottery winner

Reply:
A layer of indirection would have worked too: `public close_lottery_internal()` could return the winner and the `entry` function just calls that and ignores the return value.

That's simpler, but this is okay too.